waisntechtools/alerts/tests/test_messenger.py

```diff
@@ -0,0 +1,30 @@
+from unittest.mock import patch, mock_open, Mock
+from django.test import TestCase
+
+from alerts.messenger import Messenger, Message
+from alerts.models import Subscriber
+
+# TODO: Create Messenger test to return file name and file contents
+
+class MessengerTestCase(TestCase):
+    def setUp(self):
+        self.mock_client = Mock()
+        self.mock_subscriber = Mock()
+        self.mock_subscriber.phone_number = "+11234567890"
+
+    def test_messenger(self):
```

Comment:
The initial thought on this test was to test that the messages would be successfully sent without raising error BUT I've removed the test MessengerTestCase because if there is an issue with sending the message, the Twilio dashboard will alert the user with an error